testing/runner/Views/Main/RunSuite.cshtml

```diff
@@ -149,7 +149,6 @@
     <script src="@Url.ContentWithCacheBuster("~/node_modules/jquery.2/node_modules/jquery/dist/jquery.js")"></script>
 
     <script src="@Url.ContentWithCacheBuster("~/testing/helpers/sinon/sinon.js")"></script>
```

Comment:
Let's try to install sinon via npm and check if we could get rid of sinon script in our repo.